site/content/docs/main/restore-reference.md

```diff
@@ -86,7 +86,7 @@ metadata:
     velero.io/plugin-config: ""
     # this label identifies the name and kind of plugin
     # that this ConfigMap is for.
-    velero.io/change-pvc-node-selector: RestoreItemAction
+    velero.io/change-pvc-node: RestoreItemAction
```

Comment:
I think this change is incorrect, based on [the custom plugin configuration documentation](https://velero.io/docs/v1.5/custom-plugins/#plugin-configuration). The key should be the fully qualified name of the plugin here, which is `velero.io/change-pvc-node-selector`. That's also how the [plugin is registered](https://github.com/vmware-tanzu/velero/blob/main/pkg/cmd/server/plugin/plugin.go#L56).

So I think the actual change that should happen is that the code should be looking for the `velero.io/change-pvc-node-selector` ConfigMap in order to be consistent with documentation.

Reply:
Another option would be to change the name that the plugin is registered under, though it's less descriptive.

Reply:
@noamran change-pvc-node should suffice as it is expected to be the label in the config map, so do we need say selector (change-pvc-node-selector) specifically? For example if we check other plugins (e.g. "velero.io/change-storage-class") they don't mention selector. What do you think, should we just change the name that the plugin is registered under? Which one is the better solution?

Reply:
@pawanpraka1 I'm not sure what you mean about the other plugin not using a selector. [I see that the plugin looks up a ConfigMap with `velero.io/change-storage-class`](https://github.com/vmware-tanzu/velero/blob/main/pkg/restore/change_storageclass_action.go#L69), and if I look at the code for that function, [it's using a LabelSelector](https://github.com/vmware-tanzu/velero/blob/a841167ee0a7ac61d3f829220eb2d11cb5fea5da/pkg/restore/restic_restore_action.go#L255). It needs a ConfigMap with the labels `velero.io/plugin-config` and `<fully-qualified-plugin-name>`.

It's not enforced in code that the label name match exactly; however, it's a [documented convention](https://velero.io/docs/v1.5/custom-plugins/#plugin-configuration) that I'd rather not break on included plugins.

If it's easier for users to not change the ConfigMap's label, then changing the plugin's registered name could work, too.

Reply:
@nrb I meant to say selector in the name.

Reply:
Do we need to specify selector in the plugin name.

Reply:
@pawanpraka1 Ah, I see.

The only requirement is that the plugin and ConfigMap match, from my point of view. Updating the code to look for a new ConfigMap label may be disruptive to users, who might have existing ConfigMaps. It may be easier to simply change the plugin's registered name. Having `selector` in the name is more descriptive, and I think that is the proper name of the field in the spec, but I'm also not too worried about that precision personally.

Reply:
I will leave the decision upto you @nrb . I am fine with both the solution.